src/griffe/extensions/base.py

```diff
@@ -465,12 +468,14 @@ def load_extensions(exts: Sequence[str | dict[str, Any] | ExtensionType | type[E
         An extensions container.
     """
     extensions = Extensions()
-    for extension in exts:
+    for extension in exts or ():
         ext = _load_extension(extension)
         if isinstance(ext, list):
             extensions.add(*ext)
         else:
             extensions.add(ext)
+    # TODO: Deprecate and remove at some point?
+    extensions.add(*_load_extension("dataclasses"))  # type: ignore[misc]
```

Comment:
Should we add it first or last? Should we check if it is already added by the provided `exts`?